packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -681,6 +691,21 @@ export default class Health extends Component {
 
   _getOrphanVmSnapshots = createFilter(() => this.props.orphanVmSnapshots, this._getPoolPredicate)
 
+  _getOldSnapshots = createSelector(
+    () => this.props.snapshots,
+    () => this.props.schedules,
+    (snapshots, schedules) =>
+      Object.values(snapshots).filter(snapshot => {
+        const isOld = snapshot.snapshot_time * 1000 < THIRTY_DAYS
+        if (!isOld) return false
```

Comment:
And use your `Date.now()` here, to get a `Date.now` that really represent the actual time
E.g.
```ts
    (snapshots, schedules) => {
      const now = date.now()
      Object.values(snapshots).filter(snapshot => {
        const isOld = snapshot.snapshot_time * 1000 < now - THIRTY_DAYS
        if (!isOld) return false
```